Formula/n/neomutt.rb

```diff
@@ -58,6 +59,7 @@ def install
       --with-ncurses=#{Formula["ncurses"].opt_prefix}
       --with-ssl=#{Formula["openssl@3"].opt_prefix}
       --with-sqlite=#{Formula["sqlite"].opt_prefix}
+      EXTRA_CFLAGS=-I#{Formula["libiconv"].opt_prefix}/include
```

Comment:
Can it not be configured the usual way?

Reply:
Unfortunately there's no `with-libiconv=` currently.

Reply:
Could you make a pull request upstream to add that then, we need to work with upstream to fix this.

Reply:
I've hacked something together: https://github.com/neomutt/neomutt/pull/4085

I need to get second opinion on the change.
I'm planning another bug-fix release late next week.